app/main.py

```diff
@@ -1,8 +1,30 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons = [
+        Person(person["name"], person["age"])
+        for person in people
+    ]
+
+    for person in people:
+        current_person = Person.people[person["name"]]
+        if "wife" in person and person["wife"]:
+            wife_name = person["wife"]
+            if wife_name in Person.people:
+                current_person.wife = Person.people[wife_name]
```

Comment:
The code assumes that the wife's name exists in `Person.people`. It's important to add error handling to manage cases where a name might not be found, to prevent potential runtime errors.